### DISCUSSION TITLE
Self-hosting and e-mail problems

### DISCUSSION THREAD

**Discussion by samdakap** [simple-login/app#1499] [Self-hosting question]
I'm trying to setup SL on an Ubuntu server.
I own the domain name and followed the instructions.
Yet, I don't have a mailbox, so I'm not sure I understand which e-mail address I should put instead of ["support@mydomain.com](mailto:support@mydomain.com)" in the simplelogin.env file.

This brings so many question: 
• do I have to create a e-mail redirection in my registrar's account?
• Is everything supposed to be handled by postfix?
• Do I need a separate email hosting ? Like another postfix instance really saving my e-mails ?
• Is it ok to make the main user address be a redirect to another redirect that finally forward to a proton account? Is it useful or stupid?

Also, I'm wondering whether I'm supposed to delete the DNS servers entries provided by my registrar?

Thanks.

EDIT: all these questions arise from the fact that I've been successfully able to setup SL on my server, and it displays that it's doing the forwarding correctly, with every test email that I sent being forwarded. Yet, none of these e-mails reached my true ProtonMail mailbox...

**Comment by nguyenkims** [comment]
> • do I have to create a e-mail redirection in my registrar's account?

No this will be handled by SL.

> • Is everything supposed to be handled by postfix?

Yes, Postfix will receive the incoming emails, send it to SL containers and SL containers will then use Postfix to send emails to your mailbox.

> Do I need a separate email hosting ? Like another postfix instance really saving my e-mails ?

No you don't need.

> Is it ok to make the main user address be a redirect to another redirect that finally forward to a proton account? Is it useful or stupid?

I think having multiple redirects is redundant and actually not compatible with the SL reverse alias system.

> Also, I'm wondering whether I'm supposed to delete the DNS servers entries provided by my registrar?

If SL handles emails, you can remove the MX records.

> Yet, none of these e-mails reached my true ProtonMail mailbox...

I suggest to follow the troubleshooting guide on https://github.com/simple-login/app/blob/046748c443dd42df6e8d7772b4e651cb77955231/docs/troubleshooting.md#L7-L6 to check if you can send emails from your server. Sometimes the cloud service or your ISP can block sending emails.
